src/components/pageNav/index.jsx

```diff
@@ -25,7 +24,6 @@ class PageNav extends React.PureComponent {
     const { allQuestionsAnswered: prevAllQuestionsAnswered } = prevProps;
 
     if (allQuestionsAnswered && !prevAllQuestionsAnswered) {
```

Comment:
left part of this block in place to hide the required notification if it is being shown when the user completes all the questions on the page.